docs/sources/alerting/alerting-rules/_index.md

```diff
@@ -1,6 +1,5 @@
 ---
 aliases:
-  - old-alerting/create-alerts/
```

Comment:
We don't tend to remove aliases, but I think in this case it makes sense since the content has been long deprecated and slated for removal.